nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/LogFeature.java

```diff
@@ -71,6 +72,18 @@ public void setConsoleLogThreshold(String level) {
         hiddenAppender = newAppender;
     }
 
+    /**
+     * Sets a logger's level.
+     * 
+     * @param loggerName the logger name to create or to update, it can be empty or null in this case the root logger's
+     *            will be taken
+     * @param level the leven to set
+     * @since 11.1
+     */
+    public void setLevel(String loggerName, String level) {
+        Configurator.setLevel(loggerName, Level.toLevel(level));
```

Comment:
Do you know if there's side effect in other tests as we don't set back the original level?

Reply:
Yep, i asked my self the same question :), i was thinking to take time to more investigate about the way to do it like for example : add new annotation that provide the new and the original values  and process it on Before/After test to keep the original value. But i told my self this is will be the discussion of the PR ;) in addition to that the class [LogFeature.java](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/LogFeature.java) did the same thing by providing the methods `hideErrorFromConsoleLog / hideWarningFromConsoleLog` and `restoreConsoleLog`  which we should call on our test (there is no automatic restore). That why i didn't go further on the behaviour of setting the level and i let the called to do it if it's needed. 

i am open to rework and provide an automatic way to set / restore the value of the logger.

Reply:
Talked in real, we'll try to provide an annotation handled by `LogFeature` to change the level of a logger for a specific test class or method.

Reply:
Cool, i will implement it then 👍